Userland/Utilities/js.cpp

```diff
@@ -425,6 +426,18 @@ static void print_data_view(JS::Object const& object, HashTable<JS::Object*>& se
     out(" @ {:p}", data_view.viewed_array_buffer());
 }
 
+static void print_temporal_time_zone(JS::Object const& object, HashTable<JS::Object*>& seen_objects)
+{
+    auto& time_zone = static_cast<JS::Temporal::TimeZone const&>(object);
+    print_type("Temporal.TimeZone");
+    out("\n  identifier: ");
+    print_value(JS::js_string(object.vm(), time_zone.identifier()), seen_objects);
```

Comment:
What is the purpose of creating a temporary `JS::PrimitiveString` here just for passing it to `print_value()`?

Reply:
Just so that we get default string printing behavior in `print_value` (added quotes and color):

![image](https://user-images.githubusercontent.com/19366641/124677888-4867a480-deb9-11eb-8b49-ab4cabeaa692.png)

I think in the REPL that's OK for now - could be avoided if we were to move that functionality into a separate function (`print_string` or similar).

Reply:
Ahh, got it!